### PR TITLE
Add initrd addons to the UKI spec

### PR DESCRIPTION
**Pull request by DaanDeMeyer**
Corresponding systemd PR: https://github.com/systemd/systemd/pull/34680

### CODE REVIEW

File: specs/unified_kernel_image.md [hunk 1]
```diff
@@ -90,12 +90,14 @@ PE Addons must contain at least one section of the following types:
...
 * `.dtb`
 * `.ucode`
+* `.initrd`
 
 PE Addons should be sorted by their filename, and applied in this order. In case of `.cmdline` all command
 lines provided by addons are suffixed in this order to any command line included in the UKI. In case of
 `.dtb` any such section included in the UKI shall be applied first, and those provided by add-ons should then
 by applied in order as a fix-up. In case of `.ucode` the contained `cpio` archives should be prefixed to the
-regular initrds passed to the kernel, in reverse order.
+regular initrds passed to the kernel, in reverse order. In case of `.initrd` the contained `cpio` archives
```
Comment (behrmann) [inline specs/unified_kernel_image.md:99]: Since this already mentions the order of microcode files, we should probably also document the order of initrds, like in [this comment](https://github.com/systemd/systemd/pull/34680/files#diff-81e38c967437a45783a2a228a2f937dc8fb7859c3805c1b5d049a2708ffa4b7dR1220), though this is outside of the scope of this PR.

**Review by behrmann — APPROVED** [review]

**Review by Foxboron — APPROVED** [review]
